sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/permissions/PermissionRequestActivity.java

```diff
@@ -67,21 +80,20 @@ public void onCreate(Bundle savedInstanceState) {
 
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
             AppCenterLog.error(LOG_TAG, "Android version incompatible.");
-            complete(new Result(false, null));
+            complete(new Result(false, new Exception("Android version incompatible.")));
```

Comment:
"There is no need to request permissions in runtime on Android earlier than 6.0"